src/content/docs/journey-large-logs/get-started.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: How to manage large log ingest 
```

Comment:
This feels a bit flat. Perhaps ask SVG or one of the blog folks to play with some alternate titles?